src/core/prometheus.cc

```diff
@@ -660,9 +660,9 @@ class metrics_handler : public handler_base  {
 
     future<std::unique_ptr<httpd::reply>> handle(const sstring& path,
         std::unique_ptr<httpd::request> req, std::unique_ptr<httpd::reply> rep) override {
-        sstring metric_family_name = req->get_query_param("name");
-        bool show_help = req->get_query_param("help") != "false";
+        sstring metric_family_name = req->get_query_param("__name__");
```

Comment:
This change looks good to me, but If I understand correctly, this change breaks the example in prometheus.md, so that should be fixed.
When you say that "this is aligned with Prometheus naming convention", can you please add a link, or a short explanation, to which convention you are referring to? I don't see anything in https://prometheus.io/docs/practices/naming/#labels that seems to correspond to this case.

Reply:
I meant that using `__name__` for a metric name is more correct, see for example:
https://prometheus.io/docs/prometheus/latest/querying/basics/

Reply:
Now I see. Please add this link to the commit message.
Anyway, it doesn't mean that `__name__` is more correct - you don't actually implement the prometheus query syntax (e.g., you don't have the `=~` syntax) - but it's good for inspiration.